packages/headless/src/api/search/search/product-recommendation.ts

```diff
@@ -1,6 +1,6 @@
 export interface ProductRecommendation {
   /**
-   * Document Uri in the index. Useful for analytics.
+   * Deprecated — has no impact.
```

Comment:
Should we use the `@deprecated` annotation so that vscode crosses the name out if trying to specify it?

Would also suggest a more descriptive wording:
"This key is no longer used by the Coveo platform. There is no need to send it"

Reply:
Good point 👍 
```suggestion
   * @deprecated This parameter is no longer used by the Coveo platform. There is no need to provide it.
```